gef-ui/server/controllers/amendments/cover-end-date/validation.ts

```diff
@@ -0,0 +1,48 @@
+import { add, isAfter, isBefore, startOfDay, getUnixTime } from 'date-fns';
+import { applyStandardValidationAndParseDateInput, DayMonthYearInput, COVER_END_DATE_MAXIMUM_YEARS_IN_FUTURE } from '@ukef/dtfs2-common';
+import { ErrorsOrValue } from '../../../types/errors-or-value';
+import { mapValidationError } from '../../../utils/map-validation-error';
+
+export const validateAndParseCoverEndDate = (dayMonthYear: DayMonthYearInput, coverStartDate: Date): ErrorsOrValue<number> => {
```

Comment:
FWIW we can use an alias for `number` to make it explicit what this is returning
```suggestion
export const validateAndParseCoverEndDate = (dayMonthYear: DayMonthYearInput, coverStartDate: Date): ErrorsOrValue<UnixTimestampSeconds> => {
```